src/main/java/net/pms/configuration/PmsConfiguration.java

```diff
@@ -243,6 +243,7 @@
 	protected static final String KEY_PLUGIN_DIRECTORY = "plugins";
 	protected static final String KEY_PLUGIN_PURGE_ACTION = "plugin_purge";
 	protected static final String KEY_PRETTIFY_FILENAMES = "prettify_filenames";
+	@Deprecated
```

Comment:
Cool, yeah that is what I meant, also I wouldn't rename it to `OLD` anymore since that's what the deprecated tag is there for